qemu/tests/boot_with_disable_ept.py

```diff
@@ -0,0 +1,29 @@
+from avocado.utils import process
+
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Boot guest after disable ept/npt:
+    1) Disable ept/npt
+    2) Boot up guest
+    3) Destory the guest and restore ept/npt
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    def check_parameter_status():
+        output = process.getoutput(params["check_status_cmd"])
+        if output != params["expected_status"]:
+            test.fail("Disable %s failed" % params["parameter"])
+
+    check_parameter_status()
```

Comment:
I suppose here we don't need to define this function, the total `run()` function is very short and we don't use this function any other where.